contracts/FlowCallbackScheduler.cdc

```diff
@@ -388,7 +389,7 @@ access(all) contract FlowCallbackScheduler {
         }
 
         /// Get all timestamps that are in the past (less than or equal to current timestamp)
-        access(all) fun past(current: UFix64): [UFix64] {
+        access(all) fun getTimestampsBefore(current: UFix64): [UFix64] {
```

Comment:
I actually preffer having more concise names, and if something is on the type it doesn't have to repeat the type, so it's `getBefore` becasue you would anyway have it called like: `pastTimestamps.getBefore()` while `pastTimestamps.getTimestampsBefore()` just repeats the type